spec/models/moab_storage_root_spec.rb

```diff
@@ -19,14 +19,31 @@
     expect(moab_storage_root).to be_valid
   end
 
-  it 'enforces unique constraint on name (model level)' do
-    expect do
-      described_class.create!(moab_storage_root.attributes.slice('name', 'storage_location'))
-    end.to raise_error(ActiveRecord::RecordInvalid)
+  describe 'enforces unique constraint on name' do
```

Comment:
What would you think about re-organizing the tests to look more like the following:

```ruby
  describe 'name attribute' do
    context 'at model level' do
      it 'must be unique' do
        msg = 'Validation failed: Name has already been taken'
        expect do
          described_class.create!(name: 'storage-root-01', storage_location: '/storage_root02')
        end.to raise_error(ActiveRecord::RecordInvalid, msg)
      end
    end

    context 'at db level' do
      it 'must be unique' do
        expect { moab_storage_root.dup.save(validate: false) }.to raise_error(ActiveRecord::RecordNotUnique)
      end
    end
  end
```

I'm thinking mostly about the readability conventions around the strings that go with `describe`, `context`, and `it` blocks. Not a big concern, admittedly.

Reply:
Done; just pushed up commit with suggested changes.